backend/devflow-ai/src/main/kotlin/com/devflow/ai/service/repository/RepositoryAiBriefingService.kt

```
@@ -0,0 +1,261 @@
+                .trim()
+                .removePrefix("```json")
+                .removePrefix("```")
+                .removeSuffix("```")
+                .trim()
+
+            logger.debug("AI 응답 파싱 시작: 길이=${cleanedResponse.length}")
+
+            // JSON 파싱 (DailyBriefing 도메인 엔티티의 내부 타입 사용)
+            return objectMapper.readValue(cleanedResponse, ParsedBriefing::class.java)
+
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)